pom.xml

```diff
@@ -17,7 +17,7 @@
 	</parent>
 
 	<properties>
-		<java.version>8</java.version>
+		<java.version>18</java.version>
```

Comment:
Is this really needed? I have the main branch working with Java 11

Reply:
on my machine, [this Path.of](https://github.com/camunda-community-hub/camunda-8-lowcode-ui-template/blob/98effcc309b3c43ec6c83728f3dd5e9e1c99894f/src/main/java/org/example/camunda/process/solution/service/FormService.java#L35) syntax is failing because java 8 doesn't support it.

So I think we should bump java version up to at least support the syntax that's used. But, fair enough, probably doesn't need to be version 18